code/modules/gear_presets/uscm.dm

```diff
@@ -52,6 +55,17 @@
 			var/obj/item/device/radio/headset/almayer/marine/equipped_headset = H.wear_r_ear
 			equipped_headset.add_hud_tracker(H)
 
+/datum/equipment_preset/uscm/load_rank(mob/living/carbon/human/H)
```

Comment:
no one letter vars

Reply:
It's like that everywhere else in the file, for the same proc.

Reply:
do it anyway